src/pages/components/button/usage.mdx

```diff
@@ -102,186 +132,419 @@ import { Add16 } from "@carbon/icons-react";
+</Column>
+</Row>
 
 <Row>
-<Column colLg={8}>
+<Column colSm={2} colMd={4} colLg={4}>
 
-![Secondary/Primary button order in Modals](images/button-usage-4.png)
+#### 1. Contained button
 
-<Caption>Secondary / Primary button order in a modal</Caption>
+A. Text label <br />
+B. Container <br />
+C. Icon (optional)
+
+#### 3. Ghost button
+
+A. Text label <br />
+C. Icon (optional)
+
+</Column>
+<Column colSm={2} colMd={4} colLg={4}>
+
+#### 2. Outlined button
+
+A. Text label <br />
+B. Container <br />
+C. Icon (optional)
+
+#### 4. Icon button
+
+B. Container (optional) <br />
+C. Icon
 
 </Column>
 </Row>
 
-## Button alignment
+<DoDontRow>
+  <DoDont caption="Do left-align text in a button, even if the button is wide.">
 
-Button alignment depends on where the buttons appear.
+![Do left-align text in a button, even if the button is wide.](images/button_usage_5.png)
 
-- Use full-bleed or full-justified buttons for modals and side panels.
-- Use left-justified buttons for forms on their own page (not in a modal or side panel).
-- Use right-justified buttons if the primary response advances the user through a multistep configuration such as a wizard.
+  </DoDont>
+  <DoDont type="dont" caption="Do not center text in a button, even if the button is wide.">
 
-### Modal button alignment
+![Do not center text in a button, even if the button is wide.](images/button_usage_6.png)
 
-Single buttons or button groups in modals use full-bleed, full-width buttons. Single buttons span the full width of the modal.
+  </DoDont>
+</DoDontRow>
 
-Full-width buttons provide significant advantages in time-to-target evaluations using [Fitts’ Law](https://www.interaction-design.org/literature/book/the-glossary-of-human-computer-interaction/fitts-s-law), and are increasingly common in other contexts users are likely to encounter.
+### Button size
+
+Buttons are named according to their size and usage in the Carbon Sketch kit and in the code. The table below adds more context around the use case for each size.
 
 <Row>
 <Column colLg={8}>
 
-![Primary button in a modal](images/button-usage-3.png)
-
-<Caption>Single primary button in a modal</Caption>
+![Button sizes](images/button_usage_3.png)
 
 </Column>
 </Row>
 
+| Name         | Height (px/rem) | Use Case                                                                           |
+| ------------ | --------------- | ---------------------------------------------------------------------------------- |
+| _Full bleed_ | 64/4            | Use when buttons bleed to the edge of a component, like in side panels and modals. |  |
```

Comment:
I specifically do not want to remove this because it is a part of a coded component that we offer and it's widely seen in side panels all over product. I changed it to this instead of removing: 

"Use when buttons bleed to the edge of a larger component, like side panels or modals. This variant is not offered in the kit because it does not currently have a coded counterpart. Carbon only offers it in the context of the [modal component](components/modal/usage)."